xbmc/utils/ExecString.cpp

```diff
@@ -108,7 +111,7 @@ void SplitParams(const std::string& paramString, std::vector<std::string>& param
           if (quotaPos > 1 && quotaPos < parameter.length() - 1 && parameter[quotaPos - 1] == '=')
           {
             parameter.erase(parameter.length() - 1);
-            parameter.erase(quotaPos);
+            parameter.erase(quotaPos, 1);
```

Comment:
please drop

Reply:
That's needed - it's a bug. That part of ExecString seems to exist to remove the quotes, but without the ,1 it removes the entire rest of the string.

Reply:
Or would you like this in a seperate PR??